en_US/durability/management.md

```diff
@@ -1,13 +1,13 @@
-# Configure and Manage Durable Storage
+# Configure and Manage Durable Sessions
 
-This document provides references and instructions for configuring, managing, and optimizing durable storage within EMQX, including sessions and storage configuration.
+This document provides references and instructions for configuring, managing, and optimizing durable sessions within EMQX, including sessions and storage configuration.
 
 ## Configuration Parameters
 
-Configuration for durable storage is divided into 2 main categories:
+Configuration for durable sessions is divided into 2 main categories:
 
-- `durable_sessions`: Contains settings related to MQTT clients' sessions, including how they consume data from durable storage and data retention parameters.
-- `durable_storage` Manages the settings of the durable storage system holding the MQTT message data.
+- `durable_sessions`: Contains settings related to MQTT clients' sessions, including how they consume data from durable sessions and data retention parameters.
+- `durable_storage` Manages the settings of the durable sessions system holding the MQTT message data.
```

Comment:
This is not correct. Please revert. Durable storage is a _database_, it has no knowledge of sessions.

Reply:
Sorry, it was mistakenly "search & replaced" due to the feature renaming. I have reverted the wording.